File: clients/haskell/src/MyStrategy.hs
module MyStrategy (MyStrategy, createMyStrategy, getOrder) where

import Control.Monad.Trans.State.Lazy (State)
import Model.Constants
import Model.Game
import Model.Order
import Data.Int
import Data.Map
import Model.UnitOrder

data MyStrategy = MyStrategy

createMyStrategy :: Constants -> MyStrategy
createMyStrategy _ = MyStrategy

getOrder :: Game -> State MyStrategy Order
getOrder _ = return Order { unitOrders = empty }

File: clients/java_graal/src/main/java/ai_cup_22/MyStrategy.java
package ai_cup_22;

public class MyStrategy {
    public MyStrategy(ai_cup_22.model.Constants constants) {}
    public ai_cup_22.model.Order getOrder(ai_cup_22.model.Game game, DebugInterface debugInterface) {
        return new ai_cup_22.model.Order(new java.util.HashMap<>());
    }
    public void debugUpdate(int displayedTick, DebugInterface debugInterface) {}
    public void finish() {}
}